app/views/users/backup_code_setup/reminder.html.erb

```diff
@@ -0,0 +1,21 @@
+<% title @presenter.title %>
+
+<%= image_tag asset_url('user-signup-ial1.svg'), width: 107, height: 119, alt: '', class: 'margin-bottom-4' %>
+
+<%= render PageHeadingComponent.new.with_content(@presenter.heading) %>
+
+<p class='margin-top-1 margin-bottom-4'>
+  <%= @presenter.body_info %>
+</p>
+
+<div class="col-12">
```

Comment:
I've seen this `col-12` make an appearance quite often in recent months. `col-12` is a remnant from [Basscss](https://basscss.com/) which was removed quite a while ago. It seems redundant anyways, since it just assigns `width: 100%`, which would be the default for a block-level `<div>` element.

Can we just remove the wrapper?

```suggestion
```